### ISSUE TITLE
Not working with current replays

### ISSUE THREAD

**Issue by J-Sears** [itselectroz/brawlhalla-replay-reader#2]
Are there any plans to get this working with the current version of the game? I think this is a very interesting tool and want to try to make something using it but can't get it to work on the current replay files.

**Comment by itselectroz** [comment]
The tool should be updated for 8.04 now. There are two unknown variables in GameSettings, if you know what they are feel free to contribute.

**Comment by J-Sears** [comment]
Thank you! I still am getting an error "Error: Second version check does not match first version check" but that might be user error. I am running it through the Command Line Tool.

**Comment by itselectroz** [comment]
Just double check you’ve installed the latest version, might be worth running from source in case there’s been any publishing issues.

Feel free to upload your replay file or post your discord username and I can have a look for you.

**Comment by J-Sears** [comment]
I've made sure that I've installed the latest version and I'm still encountering the error. Here's the replay file I've been trying to open, it's just the most replay file in the folder. I might have node.js configured improperly or am not installing the command line tool properly. I appreciate all the help with this.
[[8.04] DemonIsland.zip](https://github.com/itselectroz/brawlhalla-replay-reader/files/14301759/8.04.DemonIsland.zip)
